### PR TITLE
Rework lesson landing page to reflect Amdahl's Law instead of Zipf's.

### LINKED ISSUE #4: Rework `index.md`
This can only really be tackled once we have a better idea of the whole lesson but will include
- [x] Introducing the data we will create/use
- [x] Identifying the prereqs and where they can be found (and I notice now that HPC Carpentry will not fulfil the Python prereq...so how will we tackle this?)

### PR DESCRIPTION
**Pull request by tkphd**
Closes #4.

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -4,52 +4,48 @@ root: .  # Is the only page that doesn't follow the pattern /:path/index.html
...
-
-The example workflow performs a frequency analysis of several public domain
-books sourced from [Project Gutenberg](https://www.gutenberg.org/), testing
-how closely each book conforms to [Zipf's Law][ref-zipf]. This example has
-been chosen over a more complex scientific workflow as the goal is to appeal
-to a wide audience and to focus on building the workflow without getting
+The example workflow will launch several cluster jobs with the [Amdahl][amdahl]
+program from the [Introduction to High-Performance Computing][hpc-intro] using
+different numbers of processors, collect the output from each job, and create a
+graph of "speedup" (runtime divided by the runtime with just one processor) as
```
Comment (ocaisa) [inline index.md]:
```suggestion
graph of "speedup" (reference runtime, usually one processor or node or GPU, divided by the runtime with increased compute resources) as
```

Comment (ocaisa) [reply index.md]: Speed-up (hopefully!) goes up from 1, so you need the larger number on top

Comment (tkphd) [reply index.md]: :zany_face:

**Review by reid-a — APPROVED** [review]
Looks good.

**Pull request merged**